lib/fetchers/tar.rb

```diff
@@ -28,6 +32,14 @@ def initialize(target)
       end
     end
 
+    def url
```

Comment:
Yeah, I think we can remove this for now.  This was part of an earlier organization of the code that I moved away from.  Overall, though, the goal was to eventually have the fetcher be responsible for knowing the fully resolved URL of a given target. So you could do something like `t = Inspec::Fetcher.resolve(target); t.url` and get a URL suitable to stick in a lockfile.